packages/dashboard/src/vaadin-dashboard.js

```diff
@@ -33,6 +33,63 @@ import { WidgetResizeController } from './widget-resize-controller.js';
 /**
  * A responsive, grid-based dashboard layout component
  *
+ * ### Quick Start
+ *
+ * Assign an array to the [`items`](#/elements/vaadin-dashboard#property-items) property.
+ * Set a renderer function to the [`renderer`](#/elements/vaadin-dashboard#property-renderer) property.
+ *
+ * The widgets and the sections will be generated and configured based on the renderer and the items provided.
+ *
+ * ```html
+ * <vaadin-dashboard></vaadin-dashboard>
+ * ```
+ *
+ * ```js
+ * const dashboard = document.querySelector('vaadin-dashboard');
+ *
+ * dashboard.items = [
+ *   { title: 'Widget 1 title', header: 'Widget 1 Header', rowspan: 2 },
+ *   { title: 'Widget 2 title', header: 'Widget 2 Header', colspan: 2 },
+ *   {
+ *     title: 'Section title',
+ *     items: [{ title: 'Widget in section title', header: 'Widget in Section Header'}]
+ *   },
+ *   // ... more items
+ * ];
+ *
+ * dashboard.renderer = (root, _dashboard, { item }) => {
+ *   if (!root.firstElementChild) {
+ *     root.append(document.createElement('vaadin-dashboard-widget'));
+ *   }
+ *   root.firstElementChild.widgetTitle = item.title;
```

Comment:
The renderer in the dev page isn't ideal as a usage example. Prefer something like:
```js
const widget = root.firstElementChild || document.createElement('vaadin-dashboard-widget');
if (!root.contains(widget)) {
  root.appendChild(widget);
}
widget.widgetTitle = item.title;
```

Would also not encourage using `innerHTML` for the content as it can be dangerous. Maybe setting text content for the widget would suffice (no need for the custom header content since it's not a commonly used feature).

Reply:
Updated the PR to remove the use of `innerHTML` and `header`. Added `textContent` instead.